packages/search/tests/unit/services/navi-search-provider-test.js

```diff
@@ -57,4 +59,22 @@ module('Unit | Service | navi-search-provider', function(hooks) {
     let results = await service.search.perform('something');
     assert.equal(results.length, 0, 'Returns no results');
   });
+
+  test('provider returns error response', async function(assert) {
+    assert.expect(1);
+
+    const oldError = Ember.Logger.error;
+
+    Ember.Logger.error = function(message) {
+      assert.equal(
+        message,
+        'Provider NaviSampleSearchProviderService failed to return results. [object Object]',
```

Comment:
This seems like a bad error message if an object is being cast to a string like that. Can that be cleaned up?